File: UVa10082.java
import java.util.*;

class Main {
	public static void main(String args[]) {
		Scanner sc = new Scanner(System.in);
		
		String L = "ZXCVBNM,./ASDFGHJKL;\'QWERTYUIOP[]\\`1234567890-=";
		
		while (sc.hasNext()) {
			String s = sc.nextLine();
			for (int i = 0; i < s.length(); i++) {
				switch (s.charAt(i)) {
					case ' ':
						System.out.print(" ");
						continue;
					default:
						System.out.print(L.charAt(L.indexOf(s.charAt(i))-1));
				}
			}
			System.out.println();
		}
	}
}

File: UVa10082.cpp
#include <iostream>
#include <string>
using namespace std;

int main(){
	
	string s;
	char A[]={"`1234567890-=QWERTYUIOP[]\\ASDFGHJKL;'ZXCVBNM,./"};

	while(std::getline(std::cin,s))
	{
		for(int i=0;i<s.size();i++)
		{
			if(s[i]==' ')
					cout << " ";
			for(int j=0; A[j] != '\0'; j++)
				if(s[i]==A[j])
					cout << A[j-1];
		}
		cout << endl;
	}
	
}
